website/util/__init__.py

```diff
@@ -58,7 +57,7 @@ def api_url_for(view_name, _absolute=False, _xml=False, *args, **kwargs):
 def api_v2_url(path_str,
                params=None,
                base_route=website_settings.API_DOMAIN,
-               base_prefix=api_settings.API_PREFIX,
+               base_prefix='',
```

Comment:
Perhaps it would be helpful for one of the unit tests to cover the new domain expected in deployment, with the new default value for base_prefix? (eg change `test_api_v2_url_with_params`)